docs/content/en/docs/topics/provision-a-cluster/_index.md

```diff
@@ -374,7 +374,10 @@ Notice the value of `size: 1` in cassdc.yaml. This is the Cassandra DataCenter d
 
 To scale up, you could change the `size` to 3. Example with helm:
 
-`helm upgrade k8ssandra-cluster-a k8ssandra/k8ssandra-cluster --set size=3`
+`helm upgrade k8ssandra-cluster-a k8ssandra/k8ssandra-cluster --set size=3 --reuse-values`
+
+Note: using `--reuse-values` to ensure keeping settings from previous `helm upgrade`.
```

Comment:
I still feel odd that we have to tell people how to use helm, yet I completely think we should. But I wonder, if someone executes the commands in different order, will this bite them later? Say they scaled first and then modified the traefik rules.

Reply:
>I still feel odd that we have to tell people how to use helm

Why is that? I did not have much prior experience with Helm prior to this project. I have been bitten by this very thing :)

>I wonder, if someone executes the commands in different order, will this bite them later? Say they scaled first and then modified the traefik rules.

At least in theory it should not bite them. The various controller involved should making the specified state match the actual state.